packages/lib-react-components/src/Markdownz/Markdownz.spec.js

```diff
@@ -107,8 +109,8 @@ describe('<Markdownz />', function () {
         const baseURI = 'https://classrooms.zooniverse.org'
         const expectedReturnValue = `${baseURI}/talk/search?query=%23${resource}`
         wrapper.setProps({ baseURI })
-        wrapper.instance().buildResourceURL('tigers', '#')
-        expect(buildResourceURLSpy).to.have.been.calledOnce()
+        wrapper.instance().buildResourceURL(resource, '#')
+        expect(buildResourceURLSpy.withArgs(resource, '#')).to.have.been.calledOnce()
```

Comment:
Here too. My reading of the code is that `buildResourceURLSpy` spies on `wrapper.instance().buildResourceURL`. I feel like I'm missing something.